get_sbu_data.py

```diff
@@ -0,0 +1,167 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import folium
+from folium.plugins import FastMarkerCluster
+from streamlit_folium import st_folium
+from ECHO_modules.get_data import get_echo_data # Import the get_echo_data function, which is the function that does the work of retrieving data from the SBU database
+from ECHO_modules.get_data import get_spatial_data # Import this function, which will help us get county boundaries
+from ECHO_modules.geographies import spatial_tables, fips, region_field, states # Import for mapping purposes
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+# Styles for States ("other") and selected regions (e.g. Zip Codes) - "this"
+map_style = {'this': {'fillColor': '#0099ff', 'color': '#182799', "weight": 1},
+'other': {'fillColor': '#FFA500', 'color': '#182799', "weight": 1}}
+
+selected_state = st.selectbox(
+    'Select a state',
+    states)
+
+# Get list of counties for selected state
+url = "https://raw.githubusercontent.com/edgi-govdata-archiving/"
+url += "ECHO_modules/packaging/data/state_counties_corrected.csv"
+all_counties = pd.read_csv( url )
+counties = all_counties[all_counties['FAC_STATE'] == selected_state]['County']
+counties = counties.unique()
+
+selected_county = st.selectbox(
+    'Select a county',
+    counties)
+
+st.write('You selected:', selected_county, ", ", selected_state)
+
+def format_county(selected_county):
+    '''
+    Returns the formatted county string from uppercase to lowercase and capitalized
+    Example: format_county('SAN FRANCISCO') returns 'San Francisco'
+    '''
```

Comment:
My suggestion here is to have the function spec be more insightful and keeps the parameter name consistent. The spec is a bit confusing without the example. Maybe reword it to be "Returns the title capitalization of the selected_county".